lms/djangoapps/onboarding/models.py

```diff
@@ -901,3 +901,51 @@ class RegistrationType(models.Model):
         related_name='registration_type'
     )
     choice = models.SmallIntegerField(default=1, null=False)
+
+
+class Education(models.Model):
+    """
+    Model to store user education information
+    """
+    linkedin_id = models.IntegerField(null=True, blank=True, unique=True)
```

Comment:
Please make sure we will get linkedin_id in integer not string

Reply:
As per documentation they are catered as an integer.
https://docs.microsoft.com/en-us/linkedin/shared/api-guide/concepts/urns?context=linkedin/context#ids

Reply:
done